models/games.model.js

```diff
@@ -162,3 +162,7 @@ exports.removeCommentById = (comment_id) => {
       }
     });
 };
+
+exports.selectEndpoints = () => {

```

Comment:
This is really great, but you should do this in the controller as it isn't connected to the database 👍